src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java

```diff
@@ -176,20 +226,25 @@ private static void createUser(final String userName) throws SQLException {
         statement.execute("GRANT ALL PRIVILEGES TO " + userName + "");
     }
 
-    @Test
-    void testSelectFromExaRlsUsersThrowsException() {
-        assertThrows(SQLException.class, () -> statement.execute("SELECT * FROM EXA_RLS_USERS"));
+    @ParameterizedTest
+    @ValueSource(strings = { VIRTUAL_SCHEMA_RLS_JDBC_NAME, VIRTUAL_SCHEMA_RLS_JDBC_LOCAL_NAME,
```

Comment:
Use a method source instead to reduce redundancy.

Reply:
done